_src/_posts/2017-04-24-learning-rtb.markdown

```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title:  Learning from the RTB market
-date:   2017-04-24
+date:   2017-01-31
```

Comment:
@memeplex the date should be 2017-04-24 in order to keep a fake perception that the engineer blog is "more" active